tests/AgDatabaseMove.Unit/FileToolsTest.cs

```diff
@@ -0,0 +1,60 @@
+namespace AgDatabaseMove.Unit
+{
+  using System.Collections.Generic;
+  using SmoFacade;
+  using Xunit;
+
+
+  public class BackupFileToolsTest

```

Comment:
Do we want to add a couple of example failure cases here to as well to ensure full coverage?

Reply:
I have failure cases, check down below :)